appveyor.yml

```diff
@@ -31,6 +34,6 @@ artifacts:
     name: wheel
 
 deploy_script:
-  - ps: if ($env:APPVEYOR_REPO_TAG -eq $TRUE) { pip install twine; twine upload dist/* }
+  - ps: if ($env:APPVEYOR_REPO_TAG -eq $TRUE) { pip install twine; twine upload --skip-existing dist/* }
```

Comment:
At first I wondered if this should this be `dist/*.whl` as in zopefoundation/BTrees#93 , but with `--skip-existing` that shouldn't matter.

Reply:
No, it's quite intentionally left as * so that in theory people can make releases by pushing git tags without having to manually upload an sdist.